tutorials-v5/time-evolution/022_v5_paper-smesolve.md

```diff
@@ -0,0 +1,132 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: qutip-tutorials-v5
+    language: python
+    name: python3
+---
+
+# QuTiPv5 Paper Example: Stochastic Solver - Homodyne Detection
+
+Authors: Maximilian Meyer-Mölleringhof (m.meyermoelleringhof@gmail.com)
```

Comment:
Might be fair to add Neill here as well?

Reply:
Of course!
That is actually related to a more general question, whom I should put in the authors list. Next to me and everybody that reviewed it, would it make sense to include the people that wrote the specific section in the paper as well? Is is that too tedious?

Reply:
Yeah i think that is fair too. I will try and point out on each PR who provided the example.  btw, on this particular one, maybe also good to add a sentence like ''a similar example is also available in the qutip documentation, https://qutip.readthedocs.io/en/qutip-5.0.x/guide/dynamics/dynamics-stochastic.html#stochastic-master-equation"